CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing to the RGSoC Teams App
+
+First of all, thank you for your contribution to the RGSoC Teams App. Or the consideration thereof. :smile: The Teams App is a collaborative effort of [many contributors](https://github.com/rails-girls-summer-of-code/rgsoc-teams/graphs/contributors).
+
+Please note our [Code of Conduct](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/CODE_OF_CONDUCT.md).
+
+There are different ways to get involved.
+
+0. Find work that needs to be done
+1. Reporting an error
+2. Suggesting a new feature
+3. Resolving an existing issue
+
+## Find work that needs to be done
+
+How things are organized:
+
+### Labels
+
+We use labels to track, manage and organize issues and pull requests.
+
+| Label Name         | Issues                  | Description |
+| ------------------ |:-----------------------:| ------------|
+| `beginner-friendly`| [🔍][beginner]      | Less complex issues, suitable for your first steps in open source. |
+| `bug`              | [🔍][bug]           | Confirmed misbehavior of the application code. |
+| `duplicate`        | [🔍][duplicate]     | Issues which have already been reported. |
+| `in progress`      | [🔍][in progress]   | Work in progress, things which are already taken care of. |
+| `invalid`          | [🔍][invalid]       | Issues which aren't *"real"* issues (e.g. user errors). |
+| `needs review`     | [🔍][needs review]  | Work still needs to be reviewed by another member of the team. |
+| `#pairwithme`      | [🔍][#pairwithme]   | Looking for someone to pair on this. |
+| `question`         | [🔍][question]      | Feature and code related questions which are no bug reports or feature requests. |
+| `rgsoc-<year>`     | [🔍][rgsoc-<year>]  | Issues that a pair of RGSoC students of a given `<year>` working on the Teams App as their project can work on. |
+| `styling`          | [🔍][styling]       | CSS and UI related things. |
+| `wontfix`          | [🔍][wontfix]       | The team has decided to not fix these things for now, e.g. because the whole feature will be replaced soon. |
+| `would be nice`    | [🔍][would be nice] | Features and enhancement which are *"nice to have"* but not super necessary. |
+
+[beginner]:      https://github.com/rails-girls-summer-of-code/rgsoc-teams/labels/beginner-friendly
```

Comment:
I wasn't aware that you could do link-shortcuts like that in (GHF)Markdown. Very cool!